### ISSUE TITLE
Active tab is hard to tell apart when the window is unfocused

### ISSUE THREAD

**Issue by gkatev** [EliverLara/Nordic#181]
Hi, I'm using the Nordic theme, on Cinnamon, on Arch linux (installed from AUR). I've been finding it hard to quickly tell which among multiple tabs is the currently active one, when the whole window is not focused. Here's an example with Terminator:

![focused-unfocused-tabs](https://user-images.githubusercontent.com/31203804/147496148-5373c275-2d41-451f-a908-40062d8a14cd.jpg)

To be honest, even when the window *is* focused(left on the attached image), it perhaps could be more apparent that the rightmost tab is selected. I believe that in this case I don't get confused because the tab's title's color is white(-r).

However, when the window is unfocused(right on the attached image), I think it looks as if the leftmost tab "pops out", making me think that it is that one that is activated (while it's not). At 3+ tabs I usually don't have a problem, as the active one stands out.

**Comment by gkatev** [comment]
I finally looked a bit more into this. What we're seeing here is the combination of two factors.

With all software stock, this is what the situation is (left = focused, right = unfocused):

![original](https://github.com/EliverLara/Nordic/assets/31203804/9b2d363d-4183-4a91-9c6d-90446486bb13)

I find it a bit hard(-er-than-it-should-be) to tell which tab is active, especially when the window is unfocused.

The first factor is terminator itself providing some its own CSS, which overrides the background color set by the theme for the "notebook header". The offending code is in [terminator/terminatorlib/terminator.py:394](https://github.com/gnome-terminator/terminator/blob/master/terminatorlib/terminator.py#L394). Removing this `background-color` CSS rule improves the situation:
 
![terminator_fixed](https://github.com/EliverLara/Nordic/assets/31203804/2df53473-4f41-4fb5-8e08-4585bd6e9f2d)

You can see on the left (= window focused) that the color of the inactive tab changes, and there is an arguably more distinct difference between active and inactive tab. However, on the right (= window unfocused), the arguably uglier tab background color for the inactive tab is back again. I personally find this problematic because (1) the two tabs are again harder to tell apart, and (2) the balance is completely thrown off when going back and forth between the focused and unfocused states; in the focused state the active tab has a slightly lighter color, while in the unfocused it has a slightly darker one.

I find that all's better when if I remove the `background-color` rule for `notebook > header:backdrop` in [Nordic/gtk-3.0/gtk-dark.css:2978](https://github.com/EliverLara/Nordic/blob/218a1a8679fdb97aa0aa7997fdf8c5344d68fb2f/gtk-3.0/gtk-dark.css#L2978):

![all_fixed](https://github.com/EliverLara/Nordic/assets/31203804/aa0ea21e-3e81-4dcd-a9e1-329a341136ec)

With this adjustment I can tell which tab is active with more ease, whether the window is focused or not. And this change does not result in any problem recognizing a focused/unfocused window, thanks to the color, or lack thereof, of the min/max/close buttons.

If someone wants to make the same adjustments as I did above: the following snippets may be added to `~/.config/gtk-3.0/gtk.css`:
```css
/* This fixes the terminator issue, this CSS here apparently takes precedence over the app's CSS.
 * This will also affect any other app that supplies its own CSS for the color of notebook headers. */
notebook > header {
    background-color: #303643;
}

/* This makes the color of the inactive tab remain the same when the window is unfocused. Besides
 * terminator, this is also expected to affect tabs in other applications (eg. geany, nemo), resolving
 * (if you wanna call it that) the same issue as here. */
notebook > header:backdrop {
    background-color: #303643
}
```

**Comment by EliverLara** [comment]
Hi @gkatev do you think adding a bottom boder as indicator for terminator tabs would be helpful?

**Comment by gkatev** [comment]
Hi @EliverLara, I can no longer find your comment here, but I believe there could be various solutions to this problem, potentially including the bottom border you mentioned.

Personally, I'm liking the approach I outlined above, with the tab color not changing when unfocusing the window. I am finding it useful, but have only really used it for a week or so.

But also this is a more general issue, not specific only to Terminator. I've applied the same rationale to two other apps I use often (nemo, geany), which exhibited the same behaviour.

---

Geany, with stock Nordic theme (left = focused, right = unfocused):
![geany_before_x](https://github.com/EliverLara/Nordic/assets/31203804/6ad5cbe2-388f-447c-94d6-63d02030e6f9)

Geany, with the above modification:
![geany_after_x](https://github.com/EliverLara/Nordic/assets/31203804/e72b7a4c-db61-467e-a677-81066453c375)

---

Nemo, stock Nordic (left = focused, right = unfocused):
![nemo_before_x](https://github.com/EliverLara/Nordic/assets/31203804/49ea0000-1bea-4d46-a4b1-4c05a5e7d175)

Nemo, with above modification:
![nemo_after_x](https://github.com/EliverLara/Nordic/assets/31203804/ffaaf3c3-8b3e-4463-80ac-549311f5b5ed)

**Comment by gkatev** [comment]
But I see that you already integrated this in #77ca041? Plus a fix for terminator's CSS overriding the theme?

**Comment by EliverLara** [comment]
> But I see that you already integrated this in [77ca041](https://github.com/EliverLara/Nordic/commit/77ca041a171e5fadbf43fbbe571eb3fba5dbc038)? Plus a fix for terminator's CSS overriding the theme?

Yes I've adopted your suggestions, Have you tested if it looks better now?

**Comment by gkatev** [comment]
Yes, in regards to the issue I had here, with tabs being hard to tell apart, I find that it's indeed better now, thanks!

For the record, I noticed the general tab area is now "flatter" and without that many borders: (left = before, right = after):

![Screenshot_20240202_201058_2x](https://github.com/EliverLara/Nordic/assets/31203804/6eff9f58-c691-4eb4-8619-d1468f4f2b2b)

I dunno what's best, a small difference really -- I trust your creative vision.

For my  own system, I've embraced the new flatter version, and have added a slight border between the title bar and the tab bar: (left = stock Nordic, (new) flat variant, right = my modification)

![2](https://github.com/EliverLara/Nordic/assets/31203804/d7ff10dd-493a-4811-8dba-961b9c1d7623)

But anyway all of the above do look good to me! I think we're good to close this issue (?)
